homeassistant/components/onvif/config_flow.py

```diff
@@ -268,7 +262,7 @@ async def async_step_profiles(self, user_input=None):
         finally:
             await device.close()
 
-        return self.async_show_form(step_id="auth", errors=errors)
+        return await self.async_step_configure(errors=errors)
```

Comment:
We don't pass errors between steps since the errors are specific to a certain form.